src/lib/i18n/locales/en/workflow.ts

```diff
@@ -0,0 +1,9 @@
+export const Namespace = 'workflow' as const;
+
+export const Strings = {
+  terminated: 'Terminated',
+  reset: 'Reset',
+  'workflow-action-placeholder': '{{action}} from the Web UI',
+  'workflow-action-placeholder-by-email':
+    '{{action}} from the Web UI by {{email}}',
+} as const;
```

Comment:
I think we already have these strings in the `workflows` namespace, I can't think of any reason we wouldn't want to use those. Was there a reason you made a new namespace?

Reply:
I think we can put `Reset` in workflows, I think batch reset is coming soon, so it will make sense anyway